core/base/src/TObject.cxx

```diff
@@ -227,15 +227,19 @@ TObject *TObject::DrawClone(Option_t *option) const
    TObject *newobj = Clone();
    if (!newobj) return nullptr;
    if (pad) {
-      if (strlen(option)) pad->GetListOfPrimitives()->Add(newobj,option);
-      else                pad->GetListOfPrimitives()->Add(newobj,GetDrawOption());
+      if (option && strlen(option))
```

Comment:
The pattern should be
```suggestion
      if (option && option[0])
```
There's no point in looking for the null terminating car if all we want to know is whether it is somewhere other than at position 0. Can you address this in the other occurrences that you touch anyway?